repos/base-linux/src/core/io_mem_session_component.cc

```diff
@@ -12,14 +12,60 @@
  */
 
 #include <base/log.h>
+#include <linux_dataspace/client.h>
+
+#include <core_linux_syscalls.h>
 
 #include <io_mem_session_component.h>
 
 using namespace Genode;
 
+size_t Io_mem_session_component::get_arg_size(const char *args)
+{
+    size_t size = Arg_string::find_arg(args, "size").ulong_value(0);
```

Comment:
Indent with tabs

Reply:
variables should better be `const` values whenever possible, e.g., instead of `size_t size` use `size_t const size`.